packages/language-server/src/__test__/completion.test.ts

```diff
@@ -1,7 +1,7 @@
 import { TextDocument } from 'vscode-languageserver-textdocument'
```

Comment:
I don't see this test suite (`completion.test.ts`) in the output at https://github.com/prisma/language-tools/actions/runs/9094944497/job/24997090735?pr=1727#step:6:1

Am I missing something? Is it running in the tests?

Reply:
It's in there, just hidden a bit https://github.com/prisma/language-tools/actions/runs/9094944497/job/24997090735?pr=1727#step:6:66